### PR TITLE
UBI image update to 8.7-1049

### PR DESCRIPTION
**Pull request by GunaKKIBM**

### CODE REVIEW

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 1]
```diff
@@ -3,6 +3,15 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "4",
+                "release-date": "27 January 2023",
+                "cves": "CVE-2022-43680, CVE-2022-42010, CVE-2022-42011, CVE-2022-42012, CVE-2022-3821, CVE-2022-35737, CVE-2021-46848",
+                "golang": "1.18.9",
```
Comment (ashimagarg27) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json:9]: Keep `golang` field empty as in this build golang version is not getting updated

**Review by prankulmahajan — APPROVED** [review]
@mssachan Please confirm the release date. Otherwise lgtm

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 2]
```diff
@@ -3,6 +3,15 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "4",
+                "release-date": "27 January 2023",
```
Comment (ashimagarg27) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: Have `release-date` as 23 Jan

**Review by ashimagarg27 — APPROVED** [review]
PR looks good!!

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 3]
```diff
@@ -3,6 +3,15 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "4",
+                "release-date": "23 January 2023",
+                "cves": "CVE-2022-43680, CVE-2022-42010, CVE-2022-42011, CVE-2022-42012, CVE-2022-3821, CVE-2022-35737, CVE-2021-46848",
+                "golang": "",
```
Comment (mssachan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: Add current golang version for VPC block image ie. `1.18.9`

**Review by mssachan — APPROVED** [review]
lgtm

**Pull request merged**